beetsplug/lastgenre/genres-tree.yaml

```diff
@@ -156,6 +156,31 @@
     - timba
     - twoubadou
     - zouk
+- classical:
+    - ballet
+    - baroque:
+        - baroque music
+    - cantata
+    - chamber music:
+        - string quartet
+    - classic
```

Comment:
I don't think this should be here at all, TBH. :) This could be classic rock, classic punk, a generic classic, etc. Do people actually use this to tag classical music?